app/helpers/navigation_helper.rb

```diff
@@ -233,16 +234,14 @@ def add_landing_page_breadcrumb(path)
           session[:breadcrumbs] << { 'display': "#{@page.page_group.name}", 'path': path }
         end
 
-        def add_sub_page_breadcrumb
-          session[:breadcrumbs] << { 'display': "#{@page.title}", 'path': @builder_page_path }
-        end
-
```

Comment:
I think we should keep this code. It was there for instances when the user navigated from a subpage to another page. For example, If the subpage had practices, when they clicked on the card and landed on the practice's show page, they would have a breadcrumb for both the landing page and the subpage
![](https://user-images.githubusercontent.com/34111449/196284849-fd2ae73b-3ada-4eb9-b9d6-6bdbaeb29fed.png)

Reply:
Sure, I've updated the branch so it retains that method.